backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -137,6 +137,26 @@ public async Task FinishReset(string code, Stream? zipFile = null)
         await dbContext.SaveChangesAsync();
     }
 
+    public async ValueTask<Guid[]> LookupProjectOrgIds(Guid projectId)
+    {
+        var cacheKey = $"ProjectOrgsForId:{projectId}";
+        if (memoryCache.TryGetValue(cacheKey, out Guid[]? orgIds) && orgIds != null) return orgIds;
+        orgIds = await dbContext.Projects
+            .Include(p => p.Organizations)
```

Comment:
since you have an explicit select you don't need to include anything explicitly

Reply:
I assumed the Select would fail if the Include wasn't there, but I guess EF's expression compiler is smarter than that.

Reply:
Removed in commit ffd176b4.